protobuf-definitions/make.sh

```diff
@@ -20,14 +20,14 @@ mkdir -p $DST_DIR_P/$PYTHON_PACKAGE
 
 # generate proto objects in python and C#
 
-$COMPILER/protoc --proto_path=proto --csharp_out=$DST_DIR_C $SRC_DIR/*.proto
+$COMPILER/protoc --proto_path=proto --csharp_opt=internal_access --csharp_out $DST_DIR_C $SRC_DIR/*.proto
```

Comment:
makes protobufs internal visibility instead of public

Reply:
👍